### PR TITLE
Renamed Projects

### PR DESCRIPTION
**Pull request by alfiehanssen**
Project names should be in sync with README language.

Test that uploads are working in each ios example.

### CODE REVIEW

**Comment by alfiehanssen** [comment]
@nicolelehrer You have been chosen! Please review retroactively. There's no need to review the code (I didn't really change anything, just moved all of the files around in the 2 demos). Testing is just about whether the demos run and upload. Thank you! :cake:

File: Example/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload/AppDelegate.swift [hunk 1]
```diff
@@ -38,11 +39,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate
...
         application.registerUserNotificationSettings(settings)
-        
-        let cameraRollViewController = CameraRollViewController(nibName: BaseCameraRollViewController.NibName, bundle:NSBundle.mainBundle())
-        let cameraNavController = UINavigationController(rootViewController: cameraRollViewController)
 
+        let cameraRollViewController = CameraRollViewController(nibName: BaseCameraRollViewController.NibName, bundle:NSBundle.mainBundle())
         let uploadsViewController = UploadsViewController(nibName: UploadsViewController.NibName, bundle:NSBundle.mainBundle())
+        
+        let cameraNavController = UINavigationController(rootViewController: cameraRollViewController)
         let uploadsNavController = UINavigationController(rootViewController: uploadsViewController)
```
Comment (nicolelehrer) [inline Example/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload/AppDelegate.swift]: Maybe we could set the tabBarItem titles here. An item title isn't visible until its vc becomes selected.

**Comment by nicolelehrer** [comment]
@alfiehanssen still having those auth issues, right at didTapUpload in both old and new upload projects

**Comment by nicolelehrer** [comment]
@alfiehanssen works great! old and new are workin wit it :+1:

**Pull request merged**